.github/workflows/nightly.yml

```
@@ -847,7 +865,7 @@ jobs:
         with:
           token: ${{ secrets.ACTION_MONITORING_SLACK }}
   PECL:
-    if: github.repository == 'php/php-src' || github.event_name == 'workflow_dispatch'
+    if: (github.repository == 'php/php-src' || github.event_name == 'workflow_dispatch') && (github.event_name != 'workflow_dispatch' || inputs.job == '*' || inputs.job == 'PECL')
```

Can be simplified to:

```suggestion
    if: (github.repository == 'php/php-src' || (github.event_name == 'workflow_dispatch' && (inputs.job == '*' || inputs.job == 'PECL'))
```